source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/td_ekinetic_lcao.h

```diff
@@ -69,8 +68,7 @@ class TDEkinetic<OperatorLCAO<TK,TR>> : public OperatorLCAO<TK, TR>
                     const Parallel_Orbitals* paraV,
                     const ModuleBase::Vector3<double>& dtau,
                     std::complex<double>* data_pointer,
-                    std::complex<double>** data_pointer_c,
-                    TR* s_pointer);
+                    std::complex<double>** data_pointer_c);
```

Comment:
'data' is meaningless, do you have a more specific name? typically, we use '_p' at the end of a variable to indicate a pointer.